test/DynamoCoreTests/Performance/PerformanceTests.cs

```diff
@@ -112,19 +196,119 @@ private void DoWorkspaceOpenAndCompare(string filePath)
 
             CheckForDummyNodes(ws2);
 
+            newEngineMemoryData = StopCollectingMemory(checkPoint);
+
             var wcd2 = new serializationTestUtils.WorkspaceComparisonData(ws2, CurrentDynamoModel.EngineController, dsExecution: false);
 
             serializationTestUtils.CompareWorkspaceModelsMSIL(wcd1, wcd2);
 
             var newEngineCompileAndExecutionTime = model.EngineController.CompileAndExecutionTime;
 
-            Console.WriteLine("Compile and Execution time old Engine={0:0.0}+{1:0.0} ms, new Engine={2:0.0}+{3:0.0} ms",
+            Console.WriteLine("Compile and Execution time old Engine={0:0.0}+{1:0.0} ms, new Engine={2:0.0}+{3:0.0} ms, hardCoded={4:0.0}",
                 oldEngineCompileAndExecutionTime.compileTime.TotalMilliseconds, oldEngineCompileAndExecutionTime.executionTime.TotalMilliseconds,
-                newEngineCompileAndExecutionTime.compileTime.TotalMilliseconds, newEngineCompileAndExecutionTime.executionTime.TotalMilliseconds);
+                newEngineCompileAndExecutionTime.compileTime.TotalMilliseconds, newEngineCompileAndExecutionTime.executionTime.TotalMilliseconds,
+                hardCodedExecutionTime.TotalMilliseconds);
             var execution = (Path.GetFileName(filePath),
                 oldEngineCompileAndExecutionTime.compileTime, oldEngineCompileAndExecutionTime.executionTime,
-                newEngineCompileAndExecutionTime.compileTime, newEngineCompileAndExecutionTime.executionTime);
+                newEngineCompileAndExecutionTime.compileTime, newEngineCompileAndExecutionTime.executionTime,
+                hardCodedExecutionTime);
             executionData.Add(execution);
+
+            CollectMemoryData(Path.GetFileName(filePath));
+        }
+
+        private void CollectMemoryData(string graph)
+        {
+            if (oldEngineMemoryData.collected.SizeInBytes != -1)
+            {
+                var memoryInfo = (graph, oldEngine: oldEngineMemoryData, newEngine: newEngineMemoryData, hardcoded: hardcodedMemoryData);
+                memoryData.Add(memoryInfo);
+            }
+        }
+
+        private void WarmUp()
+        {
+            if (!warmupDone)
+            {
+                var warmupFile = Path.Combine(TestDirectory, "core", "performance", "simple_point.dyn");
```

Comment:
did you verify this includes loading ASM? Maybe add a comment if true?

Reply:
I will add a clarifying comment.